worlds/ladx/Options.py

```diff
@@ -40,49 +53,41 @@ class Logic(Choice, LADXROption):
 
 class TradeQuest(DefaultOffToggle, LADXROption):
     """
-    [On] adds the trade items to the pool (the trade locations will always be local items)
-    [Off] (default) doesn't add them
+    Trade quest items are randomized. Each NPC takes its normal trade quest
+    item, but gives a random item.
     """
     display_name = "Trade Quest"
     ladxr_name = "tradequest"
 
 
 class TextShuffle(DefaultOffToggle):
     """
-    [On] Shuffles all the text in the game
-    [Off] (default) doesn't shuffle them.
+    Shuffles all the text in the game.

```

Comment:
```suggestion
    Shuffles all text in the game.
```
I could go either way on this one. "the" isn't necessary when you're saying "the game" afterwards